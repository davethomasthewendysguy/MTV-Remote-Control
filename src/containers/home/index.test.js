// eslint-disable-next-line no-unused-vars
import React from 'react';
import renderer from 'react-test-renderer';

import HomePage from '.';

describe('HomePage component', () => {
	it('should render successfully', () => {
		const tree = renderer.create(<HomePage counter={{ count: 0 }} dispatch={() => ({})} />).toJSON();

		expect(tree).toMatchSnapshot();
	});
});
