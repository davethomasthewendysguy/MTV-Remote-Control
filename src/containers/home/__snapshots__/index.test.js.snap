// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HomePage component should render successfully 1`] = `
Array [
  <react.svg
    height={200}
    style={
      Object {
        "display": "block",
        "margin": "auto",
      }
    }
    width={200}
  />,
  <div
    className="styles__Wrapper-czREMl elaEsK"
  >
    <button
      className="btn "
      onClick={[Function]}
    >
      -
    </button>
    <button
      className="btn "
      onClick={[Function]}
    >
      Async -
    </button>
    <small>
      0
    </small>
    <button
      className="btn "
      onClick={[Function]}
    >
      Async +
    </button>
    <button
      className="btn "
      onClick={[Function]}
    >
      +
    </button>
  </div>,
  <img
    alt="Redux Saga Logo"
    src="redux-saga.png"
    style={
      Object {
        "display": "block",
        "margin": "auto",
      }
    }
  />,
]
`;
